components/brave_wallet_ui/components/extension/confirm-transaction-panel/index.tsx

```diff
@@ -81,8 +88,12 @@ function ConfirmTransactionPanel (props: Props) {
     visibleTokens,
     transactionSpotPrices,
     gasEstimates,
+    transactionsQueueLength,
+    transactionQueueNumber,
```

Comment:
Why not infer these fields from the `pendingTransactions` list? Wondering if `transactionQueueData` is rendundant.

```
transactionsQueueLength = pendingTransactions.length
transactionQueueNumber = pendingTransactions.findIndex(tx => tx.id === transactionInfo.id)
```